bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,87 @@
+from coalib.bearlib.languages.documentation.DocumentationComment import (
+    DocumentationComment)
+from coalib.bearlib.languages.documentation.DocstyleDefinition import (
+    DocstyleDefinition)
+from coalib.bearlib.languages.documentation.DocumentationExtraction import (
+    extract_documentation)
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class DocumentationStyleBear(LocalBear):
+    LANGUAGES = {language for docstyle, language in
+                 DocstyleDefinition.get_available_definitions()}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    ASCIINEMA_URL = 'https://asciinema.org/a/7sfk3i9oxs1ixg2ncsu3pym0u'
+    CAN_DETECT = {'Documentation'}
+
+    def run(self, filename, file, docstyle: str, language: str):
+        """
+        Checks for certain in-code documentation styles.
+
+        It currently checks for the following style:::
```

Comment:
This is for the extra colon,

so it appears like

```
    It currently checks for the following style:
```

I will add a extra space I think for readability